File: examples/TeamUpdateExample.ts
import * as DropboxSign from "@dropbox/sign";

const teamApi = new DropboxSign.TeamApi();

// Configure HTTP basic authorization: api_key
teamApi.username = "YOUR_API_KEY";

// or, configure Bearer (JWT) authorization: oauth2
// teamApi.accessToken = "YOUR_ACCESS_TOKEN";

const data: DropboxSign.TeamUpdateRequest = {
  name: "New Team Name",
};

const result = teamApi.teamUpdate(data);
result.then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling Dropbox Sign API:");
  console.log(error.body);
});

File: examples/TeamUpdateExample.rb
require "json"
require "dropbox-sign"

Dropbox::Sign.configure do |config|
    config.username = "YOUR_API_KEY"
    # config.access_token = "YOUR_ACCESS_TOKEN"
end

team_update_request = Dropbox::Sign::TeamUpdateRequest.new
team_update_request.name = "New Team Name"

begin
    response = Dropbox::Sign::TeamApi.new.team_update(
        team_update_request,
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling TeamApi#team_update: #{e}"
end
